x-pack/plugins/security/server/mocks.ts

```diff
@@ -25,6 +26,38 @@ function createSetupMock() {
   };
 }
 
+function createStartMock() {
+  const mockAuthz = authorizationMock.create();
+  const mockAuthc = authenticationServiceMock.createStart();
+  return {
+    authc: {
+      apiKeys: mockAuthc.apiKeys,
+      getCurrentUser: mockAuthc.getCurrentUser,
+    },
+    authz: {
+      actions: mockAuthz.actions,
+      checkPrivilegesWithRequest: mockAuthz.checkPrivilegesWithRequest,
+      checkPrivilegesDynamicallyWithRequest: mockAuthz.checkPrivilegesDynamicallyWithRequest,
+      mode: mockAuthz.mode,
+    },
+  };
+}
+
+function createApiResponseMock<TResponse, TContext>(
```

Comment:
__note:__ it's a temporary workaround until Core provides their own helpers for mocking new ES client response (notified Core about this as well).